### PR TITLE
Add documentation in reference.md for import-macros.

### PR DESCRIPTION
**Pull request by technomancy**
I think this is the last missing piece for the 0.4.0 release!

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## ??? / ???
 
+* Add `import-macros` for more flexible macro module loading (#180)
```
Comment (jaawerth) [inline changelog.md:5]: Maybe reference #269 here instead/in addition, since it's got a more concise write-up of the intent than the thread in #180?

**Comment by jaawerth** [comment]
Looks good!

Was omitting **(Experimental)** in the reference for `import-macros` intentional? I'm guessing so, since the language for `require-macros` is also pushing `import-macros`.

**Comment by technomancy** [comment]
Oops; no, omitting the experimental warning was an oversight; I've added it back in.

**Comment by jaawerth** [comment]
:+1:

**Pull request merged**